src/orb.yml

```diff
@@ -278,7 +278,13 @@ jobs:
       - run:
           name: Build the final image on top of the builder image
           command: |
-            s2i build --incremental . \
+            if [ -z "${WITHOUT_S2I_INCREMENTAL:-}" ]; then
```

Comment:
I'm sure you've got the syntax right, but this definitely reads like:

```ruby
if WITHOUT
  run WITH
else
  run WITHOUT
end
```

Is there any way this could be made a bit clearer?

 ```ruby
if WITHOUT
  run WITHOUT
else
  run WITH
end
```

Reply:
Oh man, thanks for commenting this. I had such a hard time building this and I just figured out why. `-z` is the `unless` of the bash world!

Because of the type of if tests that bash gives you I'm not actually sure it can be rewritten as your second way. I'm happy to add some comments to help understanding though.